src/AppReducer.js

```diff
@@ -46,10 +46,12 @@ export function reducer(state, action) {
       return newState
 
     case types.CHECK_ANSWER:
-      const answer = parseInt(state.answer, 10)
+      let answer = parseInt(state.answer, 10)
+      answer = Math.floor(answer)
```

Comment:
If I am understanding this right, this will round the answer. What I would like to happen is that only division problems that naturally have a whole number answer would be the only ones presented to the user. For example: 9/3, 4/2. But never, 3/2 or 9/4. Does that make more sense?

Reply:
Yes it does, should i delete this PR and make another one with just the autofocus?